src/ui/components/ExplorerHome/ExplorerHome.js

```diff
@@ -88,77 +98,164 @@ const useStyles = makeStyles((theme) => ({
   element: {flex: 1, margin: "20px"},
   arrowInput: {width: "40%", display: "inline-block"},
   pageHeader: {color: theme.palette.text.primary},
+  credCircle: {
+    borderColor: theme.palette.blueish,
+  },
+  grainCircle: {
+    borderColor: theme.palette.warning.main,
+  },
+  participantCircle: {
+    borderColor: theme.palette.scPink,
+  },
+  grainPerCredCircle: {
+    borderColor: theme.palette.green,
+  },
 }));
 
+const CRED_SORT = deepFreeze({
+  name: Symbol("Cred"),
+  fn: (n) => n.cred,
+});
+const GRAIN_SORT = deepFreeze({
+  name: Symbol("Grain"),
+  fn: (n) => n.grainEarned,
+});
+const PAGINATION_OPTIONS = deepFreeze([50, 100, 200]);
+
 type ExplorerHomeProps = {|
   +initialView: CredGrainView | null,
+  +currency: CurrencyDetails,
 |};
 
-export const ExplorerHome = ({initialView}: ExplorerHomeProps): ReactNode => {
+export const ExplorerHome = ({
+  initialView,
+  currency: {suffix: currencySuffix, name: currencyName},
+}: ExplorerHomeProps): ReactNode => {
   if (!initialView) return null;
 
   const classes = useStyles();
   const [tab, setTab] = useState<number>(1);
   const [checkboxes, setCheckboxes] = useState({
-    participants: false,
-    organizations: false,
-    bots: false,
-    projects: false,
+    [IdentityTypes.USER]: false,
+    [IdentityTypes.ORGANIZATION]: false,
+    [IdentityTypes.BOT]: false,
+    [IdentityTypes.PROJECT]: false,
+  });
+
+  const allParticipants = useMemo(
+    () => Array.from(initialView.participants()),
+    [initialView.participants()]
+  );
+
+  const tsParticipants = useTableState(allParticipants, {
+    initialRowsPerPage: PAGINATION_OPTIONS[0],
+    initialSort: {
+      sortName: CRED_SORT.name,
+      sortOrder: SortOrders.DESC,
+      sortFn: CRED_SORT.fn,
+    },
   });
-  const {participants, organizations, bots, projects} = checkboxes;
 
-  const handleChange = (event) => {
-    setCheckboxes({...checkboxes, [event.target.name]: event.target.checked});
+  let credTimelineSummary = [];
+  const credAndGrainSummary = {
+    totalCred: 0,
+    totalGrain: 0,
+    avgCred: 0,
+    avgGrain: 0,
   };
 
-  const data = [
-    {title: "Cred This Week", value: 610},
-    {title: "Grain Harvested", value: "6,765g"},
-    {title: "Active Participants", value: 13},
-    {title: "Grain per Cred", value: "22g"},
-  ];
+  // do the summaries of the current page of participants
+  for (const participant of tsParticipants.currentPage) {
+    // add this node's cred to the summary graph
+    credTimelineSummary = participant.credPerInterval.map(
+      (total, i) => (credTimelineSummary[i] || 0) + total
+    );
 
-  const createData = (username, cred, grain, chart) => ({
-    username,
-    cred,
-    grain,
-    chart,
-  });
+    credAndGrainSummary.totalCred += participant.cred;
+    credAndGrainSummary.totalGrain += Number(participant.grainEarned);
```

Comment:
Grain should not be cast as a number. It needs to be handled by the BigInt library used in `core/ledger/grain.js`

Reply:
Done.